### PR TITLE
Fix conflict resolution mistakes

### PR DESCRIPTION
**Pull request by smlance**
Addresses https://github.com/bundler/bundler/pull/3841.

### CODE REVIEW

File: exe/bundle_ruby [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ruby
+
+Signal.trap("INT") { exit 1 }
```
Comment (segiddins) [inline exe/bundle_ruby]: this should be gone

File: lib/bundler/deployment.rb [hunk 2]
```diff
@@ -1,66 +0,0 @@
-$stderr.puts "DEPRECATION: Bundler no longer integrates with " \
-  "Capistrano, but Capistrano provides its own integration with " \
-  "Bundler via the capistrano-bundler gem. Use it instead."
-
-module Bundler
```
Comment (segiddins) [inline lib/bundler/deployment.rb]: make sure the two? other files are also deleted

File: man/bundle-install.ronn [hunk 3]
```diff
@@ -9,6 +9,7 @@ bundle-install(1) -- Install the dependencies specified in your Gemfile
                  [--jobs=NUMBER]
                  [--local]
                  [--deployment]
+		 [--force]
```
Comment (segiddins) [inline man/bundle-install.ronn]: spaces not tabs

File: lib/bundler/dsl.rb [hunk 4]
```diff
@@ -224,8 +224,7 @@ def add_git_sources
 
       # TODO 2.0 remove this deprecated git source
       git_source(:bitbucket) do |repo_name|
-        # Double quotes here make some dsl_spec.rb specs fail
-        warn_deprecated_git_source(:bitbucket, 'https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}.git')
+        warn_deprecated_git_source(:bitbucket, "https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}.git")
```
Comment (segiddins) [inline lib/bundler/dsl.rb]: this actually needs to be moved to below `repo_name ||= user_name`

File: lib/bundler/deployment.rb (hunk 2 above)
Comment (smlance) [inline lib/bundler/deployment.rb]: capistrano and vlad? Pretty sure they're already gone.

File: lib/bundler/dsl.rb [hunk 5]
```diff
@@ -224,10 +224,9 @@ def add_git_sources
 
       # TODO 2.0 remove this deprecated git source
       git_source(:bitbucket) do |repo_name|
-        # Double quotes here make some dsl_spec.rb specs fail
-        warn_deprecated_git_source(:bitbucket, 'https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}.git')
         user_name, repo_name = repo_name.split "/"
         repo_name ||= user_name
+        warn_deprecated_git_source(:bitbucket, "https://#{user_name}@bitbucket.org/#{user_name}/#{repo_name}.git")
```
Comment (indirect) [inline lib/bundler/dsl.rb]: lol "TODO 2.0 remove this" sounds like a good plan. :)

**Pull request merged**